clipmenud

```diff
@@ -13,9 +12,10 @@ CM_MAX_CLIPS_THRESH=$(( CM_MAX_CLIPS + 10 ))
 : "${CM_SELECTIONS:=clipboard primary}"
 read -r -a selections <<< "$CM_SELECTIONS"
 
-major_version=6
-cache_dir=$CM_DIR/clipmenu.$major_version.$USER/
+cache_dir=$(clipctl cm-dir)
 cache_file=$cache_dir/line_cache
+status_file=$cache_dir/status
+echo "enabled" > $status_file
```

Comment:
RHS needs quoting (in general, you'll benefit from running `shellcheck` before submission :-))

Reply:
Added quoting. Why is quoting needed here actually?

Also, is there something misspelled? :)